src/styles/blocks/recommended.scss

```diff
@@ -0,0 +1,14 @@
+.recommended {
+  &__products {
+    @include pageGrid;
+    row-gap: 40px;
```

Comment:
Consider using a variable for '40px' to maintain consistency and ease of modification.